website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+
+[move to insurance section] By 20XX, will a major insurance company [scope of major required] in the United States offer a reduced premium for families with genetically edited children to resist common genetic diseases?
+
+Will the World Health Organization (WHO) [OR other organization] establish a special division [log of current divisions or sub-organizations required] by 20XX to monitor and advise on ethical issues related to human genetic enhancement?
+
+By 20XX, will a Nobel Peace Prize [OR other award, catalog of awards required] be awarded to an organization that works to ensure equitable access [outcome or effort based] to gene therapy [OR editing] for populations in developing countries [scope of developing nations required]?
+
+By 20XX, will at least one country have established a "genetic enhancement tax" aimed at subsidizing gene-editing or therapy [scope of interventions required] treatments or enhancements for certain populations [scope of populations required]?
+
+By 20XX, will a court in a major jurisdiction [scope of jurisdiction required] rule that children born with genetic enhancements have the right to know the specifics of their genetic modifications?
+
+
+## Technological Advancements & Clinical Applications
+
+Will CRISPR-based [OR other DNA-technology, e.g. TALENs] treatment [more specific on treatment, need divide for gene-therapy v. edits] for genetic diseases be available in over XX% of countries by 20XX?
+
+Will a genetic editing technology more precise [definition and measurement of precision required; possible shift to continuous question, "_how much_"] than CRISPR be widely used for human treatments by 20XX?
+
+Will polygenic screening for complex diseases [diseases required, e.g. heart disease, diabetes, ...] become a standard [definition of standard required] part of prenatal care by 20XX?
+
+Will gene-editing therapies for neurodegenerative diseases [disease required, e.g. Alzheimer's, Parkinson's] be approved [scope of approval required] by the FDA [organization(s) approving required] by 20XX?
+
+Will human organs [scope of organs required; enhanced organ function or treatment for organ inadeqaucy?] grown from genetically modified cells be widely used [scope of widely used required] for transplants by 20XX?
+
+Will more than XX% of all cancer therapies [scope of cancer therapies required] involve some form of DNA or genetic engineering by 20XX?
+
+Will gene-editing [scope of gene-editing required] be used to treat autoimmune diseases [diseases required, e.g., Type 1 Diabetes, Rheumatoid Arthritis] in clinical practice [scope of clinical practice required] by 20XX?
+
+Will artificial intelligence be used to automatically [definition of automatically required] design personalized gene therapies [scope of gene therapies required] for patients [patient and or newborns?] by 20XX?
+
```

Comment:
thought: "Will artificial intelligence be used..." is far too vague; if matplotlib-tier machine learning techniques count, it's basically trivial. so, suggestion: "Will gene therapy services offer personalized treatments developed by foundation models and/or expert systems with minimal human input..."